tests/rkt-monitor/main.go

```diff
@@ -96,81 +105,132 @@ func runRktMonitor(cmd *cobra.Command, args []string) {
 	}
 
 	var execCmd *exec.Cmd
-	if podManifest {
-		execCmd = exec.Command("rkt", "run", "--pod-manifest", args[0], "--net=host")
-	} else {
-		execCmd = exec.Command("rkt", "run", args[0], "--insecure-options=image", "--net=host")
-	}
-	if flagShowOutput {
-		execCmd.Stdout = os.Stdout
-		execCmd.Stderr = os.Stderr
-	}
-	err = execCmd.Start()
-	if err != nil {
-		fmt.Printf("%v\n", err)
-		os.Exit(1)
-	}
+	var loadAvg *load.AvgStat
+	var containerStarting, containerStarted, containerStopping, containerStoped time.Time
 
-	c := make(chan os.Signal, 1)
-	signal.Notify(c, os.Interrupt)
-	go func() {
-		for range c {
-			err := killAllChildren(int32(execCmd.Process.Pid))
-			if err != nil {
-				fmt.Fprintf(os.Stderr, "cleanup failed: %v\n", err)
-			}
-			os.Exit(1)
+	records := [][]string{{"Time", "PID name", "PID number", "RSS", "CPU"}}             // csv headers
+	summaryRecords := [][]string{{"Load1", "Load5", "Load15", "StartTime", "StopTime"}} // csv summary headers
+
+	for i := 0; i < repetitionNumber; i++ {
+		containerStarting = time.Now()
+
+		if podManifest {
+			execCmd = exec.Command("rkt", "run", "--pod-manifest", args[0], "--net=default-restricted")
+		} else {
+			execCmd = exec.Command("rkt", "run", args[0], "--insecure-options=image", "--net=default-restricted")
 		}
-	}()
 
-	usages := make(map[int32][]*ProcessStatus)
+		if flagShowOutput {
+			execCmd.Stdout = os.Stdout
+			execCmd.Stderr = os.Stderr
+		}
 
-	timeToStop := time.Now().Add(d)
-	for time.Now().Before(timeToStop) {
-		usage, err := getUsage(int32(execCmd.Process.Pid))
+		err = execCmd.Start()
 		if err != nil {
-			panic(err)
-		}
-		if flagVerbose {
-			printUsage(usage)
+			fmt.Printf("%v\n", err)
+			os.Exit(1)
 		}
+		containerStarted = time.Now()
```

Comment:
For better measurements, you may want to move this before the error checking branch.